include/ibamr/IBStandardForceGen.h

```diff
@@ -92,6 +92,12 @@ class IBStandardForceGen : public IBLagrangianForceStrategy
                                      const SpringForceFcnPtr spring_force_fcn_ptr,
                                      const SpringForceDerivFcnPtr spring_force_deriv_fcn_ptr = nullptr);
 
+    /*!
+     * \brief Set a uniform body force that is applied on each point in the
+     * structure with the given structure_id.
+     */
+    void setUniformBodyForce(IBTK::Vector F, int structure_id, int level_number);
```

Comment:
Should we also include an interface to set a variable body force by registering a function pointer? I can add this if you want me too.

Reply:
If you want to add this, that would be good. Could use it for the uniform case too.

Reply:
BTW, is it useful to have the interface for a uniform force, even if it is implemented internally using a function pointer? If so, could we go ahead and merge this (if the interface is OK) and then add the function pointer later?

Reply:
Sure. I'll create an issue.